schedule/yast/lvm/lvm_thin_provisioning_opensuse.yaml

```diff
@@ -0,0 +1,44 @@
+---
+name: lvm_thin_provisioning
+description: >
+  Complete OS deployment with unencrypted LVM drive management. Test creates
+  2 LVM and BIOS boot partitions. Thin pool and thin lv resides on the second
+  LVM partition, where /home (XFS) is being mounted. Partitioning is validated
+  in the booted system after the installation.
+vars:
+  LVM_THIN_LV: '1'
```

Comment:
looks like we don't need this variable anymore? I can only find it in combination with 'LVM_THIN_LV' for retrieving logs in opensusebasetest. Please, double check and if it is not needed we can remove as well from variables.md

Reply:
Funny thing, I checked and it looks like we didn't need it before either :grin: